examples/two-factor-authentication.php

```diff
@@ -2,27 +2,63 @@
 
 return [
     '*' => [
-        // Allow a totp delay in seconds (gives the user some extra time after code expired)
+        /**
+         * Allow a TOTP delay (in seconds). This gives the user some extra
+         * time after the code has expired.
+         */
         'totpDelay' => null,
 
+        /**
+         * Whether 2FA should be enabled for the front-end.
+         */
         'verifyFrontEnd' => false,
+
+        /**
+         * Whether 2FA should be anbled for the back-end/control panel.
+         */
         'verifyBackEnd' => true,
 
+        /**
+         * Whether users should be forced to enable 2FA when logging into the
+         * front-end. When this is true, users that do not have 2FA enabled will
+         * not be able to access the control panel without first enabling 2FA.
```

Comment:
Should control panel be login required area? I'm not sure, because this can be anything depending on what the login protects on the front-end

Reply:
No, you're right, that sentence should be removed.